mmv1/templates/terraform/examples/scc_v2_folder_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_pubsub_topic" "scc_v2_folder_mute_config" {
```

Comment:
What is the reason for including a pubsub topic in the test?

Reply:
addressed